modules/oe_content_featured_media_field/tests/modules/oe_content_featured_media_test/oe_content_featured_media_test.info.yml

```diff
@@ -0,0 +1,11 @@
+name: OpenEuropa Content Featured Media Test
+description: Test module for Featured Media entity browser widget.
```

Comment:
Well, the test module is not JUST for the entity browser widget no?

Reply:
For now, it is since it only contains a view and and entity browser. I can update the description.